### PR TITLE
Add configurations for PDF output on Read the Docs

### PR DESCRIPTION
**Pull request by samjwu**
Sample on 5.5.1: https://rocm.docs.amd.com/_/downloads/en/docs-5.5.1/pdf/

Ref: https://www.sphinx-doc.org/en/master/usage/configuration.html

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -14,12 +14,22 @@
 # Keep capitalization due to similar linking on GitHub's markdown preview.
 shutil.copy2('../CHANGELOG.md','./CHANGELOG.md')
 
+# for PDF output on Read the Docs
+project = "ROCm Documentation"
+author = "AMD"
```
Comment (saadrahim) [inline docs/conf.py:19]: The author is AMD Inc. not just AMD

Comment (samjwu) [reply docs/conf.py]: Done in [e82fcbb](https://github.com/RadeonOpenCompute/ROCm/pull/2305/commits/e82fcbb107ad8254910d57f82abbb06613f5812d)

Do we also want to add a copyright?

Comment (saadrahim) [reply docs/conf.py]: You should add that as well.

Comment (samjwu) [reply docs/conf.py]: [add copyright to pdf](https://github.com/RadeonOpenCompute/ROCm/pull/2305/commits/a3ec07e634775023bb537b307db72b2b8ee6e5a1)

**Review by saadrahim — APPROVED** [review]

**Pull request merged**